src/Components/Navbar/nav.jsx

```diff
@@ -1,15 +1,25 @@
 import React from "react";
 
+import s from './nav.module.css'
```

Comment:
Consider using "styles" instead of "s" - it is more readable.
Applies to all the files.

Reply:
i changed all "s" variable to "style"